cookbooks/chef-server-deploy/recipes/omnibus_standalone.rb

```diff
@@ -141,7 +141,7 @@
 
 file '/var/opt/opscode/nginx/etc/addon.d/99-supermarket-credentials_external.conf' do
   content(lazy do
-    <<-EOF
+<<-EOF
```

Comment:
Looks like the cookstyle test dislikes this change:

    recipes/omnibus_standalone.rb:144:1: C: Layout/IndentationWidth: Use 2 (not -2) spaces for indentation.
 